nodejs-ipaddr.js/nodejs-ipaddr.js.spec

```diff
@@ -0,0 +1,56 @@
+# FIXME
+# Remove nodejs_symlink_deps if installing bundled module with NPM
+
+%global npm_name ipaddr.js
+%global enable_tests 0
+
+%{?nodejs_find_provides_and_requires}
+
+Name: nodejs-%{npm_name}
+Version: 1.2.0
+Release: 1%{?dist}
+Summary: A library for manipulating IPv4 and IPv6 addresses in JavaScript
+License: MIT
+URL: FIXME
```

Comment:
Substitute for  https://github.com/whitequark/ipaddr.js, npm2rpm normally does this automatically but the package.json didn't contain anything https://github.com/whitequark/ipaddr.js/blob/master/package.json...